### ISSUE TITLE
DK ghoul

### ISSUE THREAD

**Issue by Godzila1234** [WoWManiaUK/Redemption#1230]
Its the same bug i reported here  https://github.com/WoWManiaUK/Redemption/issues/1225#event-1098611344  . I finally figured out what is the exact reason for it. The bug appears only after using Death Pact and its on cooldown so after summoning the pet again and dismount the pet will ``die`` and auto ress with low hp. The bug appears on and OUT of battlegrounds. I have video as requested but its too big so i will send  screenshots on support@wow-mania.com. 
Sorry if opening new issue wasn`t need but don`t know what the policy is.
Thank you for your good work!

**Comment by fvicaria** [comment]
Sorry I am not going to investigate this for now. It is not easily reproducible. If I have more time in the future I will get back to it.
